src/ORM/Translatable/TranslatableSubscriber.php

```diff
@@ -34,19 +35,21 @@
  */
 class TranslatableSubscriber extends AbstractSubscriber
 {
+    private $uniqueIndexNameGenerator;
     private $currentLocaleCallable;
     private $defaultLocaleCallable;
     private $translatableTrait;
     private $translationTrait;
     private $translatableFetchMode;
     private $translationFetchMode;
 
-    public function __construct(ClassAnalyzer $classAnalyzer, callable $currentLocaleCallable = null,
+    public function __construct(ClassAnalyzer $classAnalyzer, UniqueIndexNameGeneratorInterface $uniqueIndexNameGenerator, $isRecursive, callable $currentLocaleCallable = null,
```

Comment:
Can you chop down *all* arguments so the constructor is more clear about the different arguments?